python/python/glide/async_commands/core.py

```diff
@@ -2720,6 +2720,59 @@ async def zmscore(
             await self._execute_command(RequestType.ZMScore, [key] + members),
         )
 
+    async def zdiff(self, keys: List[str]) -> List[str]:
+        """
+        Returns the difference between the first sorted set and all the successive sorted sets.
+        To get the elements with their scores, see `zdiff_withscores`.
+
+        When in Cluster mode, all keys must map to the same hash slot.
+
+        See https://valkey.io/commands/zdiff for more details.
+
+        Args:
+            keys (List[str]): The keys of the sorted sets.
+
+        Returns:
+            List[str]: A list of elements representing the difference between the sorted sets.
+                If the first key does not exist, it is treated as an empty sorted set, and the command returns an
+                empty list.
+
+        Examples:
+            >>> await client.zdiff("sorted_set1", "sorted_set2", "sorted_set3")
+                ["element1"]  # Indicates that "element1" is in "sorted_set1" but not "sorted_set2" or "sorted_set3".
+        """
+        return cast(
+            List[str],
+            await self._execute_command(RequestType.ZDiff, [str(len(keys))] + keys),
+        )
+
+    async def zdiff_withscores(self, keys: List[str]) -> Dict[str, float]:
```

Comment:
why do we use Dict and not Mapping?

Reply:
I'm actually not sure what we should choose, but whatever it is, it should be consistent

Reply:
Apparently typing.Dict is deprecated in favor of builtins.dict (see [here](https://docs.python.org/3/library/typing.html#typing.Dict)). Most commands use Mapping so I've switched to Mapping